test/bin/config-cli-args.t/run.t

```diff
@@ -47,7 +47,7 @@ Once again, we can see from the generated lock file that it used both repos.
 Note that you can use the '--opam-repositories' option to overwrite any local
 extension.
 
-  $ sed -i '$d' ./config-cli-args.opam
+  $ perl -i -ne 'print unless eof' ./config-cli-args.opam
```

Comment:
I don't think depending on `perl` is a good idea here. `perl` is usually not part of POSIX and as such not installed by default on some Linux distributions. We would need to add a dependency on `conf-perl` and I assume Windows users wouldn't be happy either.

Is the `sed -i''` trick not working?

Reply:
I got it working by passing the `-e` flag explicitly to `sed`